Setup project

mohamedSabry0 opened the pull request:
In this PR:
- Linters' files were added.
- The project was created using create-react-app.
- All checks are passing, and no errors with the current code.

mohamedSabry0 committed:
update readme, fix linter errors

mohamedSabry0 committed:
fix stylelint errors

CollinsTatang reviewed (commented):
Hi @mohamedSabry0, 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file.

- [ ] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [ ] Please make sure that you add the section **Key and Future Features**, including 1-3 potential features to be added.
- [ ] Please your commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly correct that in your next commit massage.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

mohamedSabry0 committed:
Update readme

- in response to review, the readme is updated with expected features, and future ones.

rustam-tolipov reviewed (approved):
Hi @mohamedSabry0, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Developer-512px-10](https://user-images.githubusercontent.com/69011963/151541305-67f4f8e4-c79e-4f9d-a6cf-2c7efc3102c6.gif)

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
